DNN Platform/Library/Common/Globals.cs

```diff
@@ -3457,42 +3547,40 @@ public static Hashtable DeserializeHashTableBase64(string Source)
             return objHashTable;
         }
 
-        /// -----------------------------------------------------------------------------
         /// <summary>
         /// DeserializeHashTableXml deserializes a Hashtable using Xml Serialization.
         /// </summary>
         /// <remarks>
         /// This is the preferred method of serialization under Medium Trust.
         /// </remarks>
-        /// <param name="Source">The String Source to deserialize.</param>
+        /// <param name="source">The String Source to deserialize.</param>
         /// <returns>The deserialized Hashtable.</returns>
-        /// -----------------------------------------------------------------------------
-        public static Hashtable DeserializeHashTableXml(string Source)
+        [Obsolete("Deprecated in v9.8.1, this API was not meant to be public and only deserializes xml with a root of 'profile', scheduled removal in v10.")]
+        public static Hashtable DeserializeHashTableXml(string source)
         {
-            return XmlUtils.DeSerializeHashtable(Source, "profile");
+            return XmlUtils.DeSerializeHashtable(source, "profile");
         }
 
-        /// -----------------------------------------------------------------------------
         /// <summary>
         /// SerializeHashTableBase64 serializes a Hashtable using Binary Formatting.
         /// </summary>
         /// <remarks>
         /// While this method of serializing is no longer supported (due to Medium Trust
         /// issue, it is still required for upgrade purposes.
         /// </remarks>
-        /// <param name="Source">The Hashtable to serialize.</param>
+        /// <param name="source">The Hashtable to serialize.</param>
         /// <returns>The serialized String.</returns>
-        /// -----------------------------------------------------------------------------
-        public static string SerializeHashTableBase64(Hashtable Source)
+        [Obsolete("Deprecated in v9.8.1, scheduled removal in v10.")]

```

Comment:
Why is this being removed in v10 instead of v11?

Reply:
Addressed in latest commit, thanks.